libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -7,27 +7,37 @@ table.calendar-table {
   user-select: none;
   outline: none !important;
   border-spacing: 0;
+  margin-bottom: size('xxs');
 
   thead {
-    padding: 0.5em;
     border-bottom: 1px solid get-color('background-color');
```

Comment:
I suggest you move the `border-bottom` declaration to the `th` rule set. `thead` is not ideal as a direct target for styling, and you get the same result because the table borders are collapsed.